### PR TITLE
DDCNL-2955 | WRIT PAYE changes

### PR DESCRIPTION
**Pull request by shikkhasingh**

### CODE REVIEW

File: app/views/paye/PayeGovernmentSpendingView.scala.html [hunk 1]
```diff
@@ -51,16 +51,27 @@ <h2 class="heading-xlarge flush--top">@Html(Messages("paye.generic.tax_year_head
 		<dd class="value total"><strong>&pound;@viewModel.totalAmount</strong></dd>
 	</dl>
 
+@if(!isWelshTaxPayer){
```
Comment (struong) [inline app/views/paye/PayeGovernmentSpendingView.scala.html:54]: Can we inverse this?  !checks are harder to read

File: conf/messages [hunk 2]
```diff
@@ -315,6 +315,9 @@ paye.ats.treasury_spending.para3=The figures in the table above are intended as
 paye.ats.treasury_spending.para4=Spending information is published by HM Treasury.
 paye.ats.treasury_spending.scottish.spending.1=Find out more about <a href="https://www.gov.scot/publications/scottish-income-tax-2019-2020/" target="blank" rel="noopener noreferrer">your tax and public spending in Scotland</a> on the Scottish Government website.
 paye.ats.treasury_spending.total=Total
+paye.ats.treasury_spending.welsh.tax.payer.para1=The figures in the table above are intended as an illustration of how taxes are spent and not as a direct link between your Income Tax, National Insurance contributions and any specific expenditure.
```
Comment (struong) [inline conf/messages:318]: Got Welsh?

File: test/views/paye/PayeGovernmentSpendViewSpec.scala [hunk 3]
```diff
@@ -26,13 +26,14 @@ import views.html.paye.PayeGovernmentSpendingView
...
-  implicit val request = PayeAuthenticatedRequest(testNino, FakeRequest("GET", "/annual-tax-summary/paye/treasury-spending"))
+  implicit val request =
+    PayeAuthenticatedRequest(testNino, FakeRequest("GET", "/annual-tax-summary/paye/treasury-spending"))
   lazy val payeGovernmentSpendingView: PayeGovernmentSpendingView = inject[PayeGovernmentSpendingView]
 
   "view" should {
     "have correct data and heading for given taxYear" in {
 
-      val view = payeGovernmentSpendingView(PayeAtsTestData.payeGovernmentSpendViewModel).body
+      val view = payeGovernmentSpendingView(PayeAtsTestData.payeGovernmentSpendViewModel, false).body
```
Comment (struong) [inline test/views/paye/PayeGovernmentSpendViewSpec.scala:36]: extract these new false/true values to descriptive variables

File: app/models/PayeAtsData.scala [hunk 4]
```diff
@@ -24,7 +24,12 @@ case class PayeAtsData(
   summary_data: Option[DataHolder],
   income_data: Option[DataHolder],
   allowance_data: Option[DataHolder],
-  gov_spending: Option[GovernmentSpendingOutputWrapper])
+  gov_spending: Option[GovernmentSpendingOutputWrapper]) {
+  def isWelshTaxPayer: Boolean =
```
Comment (struong) [inline app/models/PayeAtsData.scala:28]: Missing tests, I would also add a comment here mentioning that using `scottish_income_tax` is not a bug and we expect the field to change in the backend in 2021.

File: app/views/paye/PayeGovernmentSpendingView.scala.html (hunk 1 above)
Comment (shikkhasingh) [reply app/views/paye/PayeGovernmentSpendingView.scala.html]: Yes we can:)
done

File: conf/messages (hunk 2 above)
Comment (shikkhasingh) [reply conf/messages:318]: nope
requested

File: test/views/paye/PayeGovernmentSpendViewSpec.scala (hunk 3 above)
Comment (shikkhasingh) [reply test/views/paye/PayeGovernmentSpendViewSpec.scala]: done

File: app/models/PayeAtsData.scala (hunk 4 above)
Comment (shikkhasingh) [reply app/models/PayeAtsData.scala:30]: done

**Review by struong — APPROVED** [review]

**Review by struong — APPROVED** [review]

**Pull request merged**